plugins/vue-query-plugin.ts

```diff
@@ -0,0 +1,5 @@
+import { QueryClient, VueQueryPlugin } from '@tanstack/vue-query'
+import Vue from 'vue'
+
+const queryClient = new QueryClient()
+Vue.use(VueQueryPlugin, { queryClient })
```

Comment:
I wonder if we could use `nuxtApp.vueApp.use` instead. of this

Reply:
done in https://github.com/kodadot/nft-gallery/pull/6053/commits/93f2f949582e35afd0160f0589423c5d70dbe838